src/suricata.c

```diff
@@ -2996,7 +2996,7 @@ int InitGlobal(void) {
     return 0;
 }
 
-int main(int argc, char **argv)
+int SC_main(int argc, char **argv)
```

Comment:
Not a fan of this name. We actually want to move away from the SC prefix that is used on and off.

When this is in use, in what mode is Suricata running? Normal? Fuzz target? Something else?

Reply:
> When this is in use, in what mode is Suricata running? Normal? Fuzz target? Something else?

Normal mode.

Fuzz targets do not use this, but they could cf what is done for fuzzing mysql project : https://github.com/google/oss-fuzz/blob/master/projects/mysql-server/targets/fuzz_mysqld.cc#L152
This was the quickest way to get all initialization right for this project